README.md

```diff
@@ -65,6 +65,39 @@ To run tests in watch mode:
 npm run test:watch
 ```
 
+### Component Organization
```

Comment:
Thought it was weird that these changes are being included here, but it's because this branch was originally created from `main` and the `assembly-ui-v1` branch doesn't have these changes yet.